src/modules/blockchainApplication/explore/components/BlockchainApplicationDetails/BlockchainApplicationDetails.js

```diff
@@ -50,7 +50,7 @@ const BlockchainApplicationDetails = ({ history, location }) => {
     lastCertificateHeight,
     lastUpdated,
     logo,
-    depositedLsk = 0,
+    escrowedLSK = '0',
```

Comment:
NRTTL this change needs to be replicated for blockchain application view and remove blockchain application as well as some test and fixture files